moz_sql_parser/sql_parser.py

```diff
@@ -371,9 +374,104 @@ def to_string(instring, tokensStart, retTokens):
 ).addParseAction(to_union_call)
 
 
-SQLParser = selectStmt
+def to_create_json_call(instring, tokensStart, retTokens):
+    # ARRANGE INTO {op: params} FORMAT
+    tok = retTokens
+    name = tok[0].lower()
+
+    types = tok.types
+    option = tok.option
+    if not types:
+        types = tok[1]
+    if not option:
+        option = tok[2:4]
+        op = options_call(option)
+    if len(tok) == 2:
+        return {"name" : name, "type": types}
+    elif len(tok) == 3 or 4:
+        return {"name" : name, "type": types, "option" : op}
+    else:
+        return tok
+
+def options_call(option):
+    op = str(option)
+    if "not" and "null" in op:
+        return "not null"
+    elif "primary" in op:
+        return "primary key"
+    elif "foreign" in op:
+        return "foreign key"
+    elif "check" in op:
+        return "check"
+    elif "default" in op:
+        return op
+    elif "index" in op:
+        return "index"
+    else:
+        return None
+
+
+def to_create_table_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    if tok:
+        return {"create table" : tok.asList()}
+
+def to_table_name_call(instring, tokensStart, retTokens):
```

Comment:
You may not need this method; pyparsing should already be setting this for you.

Reply:
If I removed this method then I can't make the output string as expected. I have used this method for a column name. How should pyparsing already be setting this for us? Can you please tell me??